### DISCUSSION TITLE
Does ‘Advantage’ on the Edit’ screen mean ‘Advanced settings’?

### DISCUSSION THREAD

**Discussion by itagagaki** [1Remote/1Remote#751] [Q&A]
Or can you describe this group differently?

<img width="737" alt="image" src="https://github.com/user-attachments/assets/c81a7891-fdbd-463a-b6e1-252f670adfc2">

**Comment by majkinetor** [comment]
English translation is bad. I will get into it soon.

**Comment by VShawn** [comment]
oh...i should be `Advanced` 😢
